tests/testcase.txt

```diff
@@ -37,6 +37,22 @@ Get HTTPS & Verify Cert with a CA bundle
     ${resp}=  Get Request  httpbin  /get
     Should Be Equal As Strings  ${resp.status_code}  200
 
+Get HTTPS & Verify Cert with CA file missing
+    [Tags]  get
+    Create Session    httpbin    https://httpbin.org   verify=${CURDIR}${/}doesnotexist.pem
+    Run Keyword And Expect Error  *  Get Request  httpbin  /get
+
+Get HTTPS & Verify Cert with a Single Certificate Chain
+    [Tags]  get
+    Create Session    httpbin    https://httpbin.org   verify=${CURDIR}${/}httpbin.chain.pem
+    ${resp}=  Get Request  httpbin  /get
+    Should Be Equal As Strings  ${resp.status_code}  200
+
+Get HTTPS & Verify Cert with a Non Matching Certificate Chain
+    [Tags]  get
+    Create Session    httpbin    https://httpbin.org   verify=${CURDIR}${/}github.chain.pem
```

Comment:
Httpbin will renew their certificate in jan. 2017. The test will fail if they use a different certificate authority. We then have to update httpbin.chain.pem.
The test with the non matching certificate is expected to fail. It will fail as long as httpbin does not switch to DigiCert CA.